### ISSUE TITLE
MEvaluate["magic(4)"] doesn't work with M10 on Linux

### ISSUE THREAD

**Issue by szhorvat** [rsmenon/MATLink#74]
With MATLink 1.0 on Mathematica 10.0.0 & MATLAB R2013b,

```
MEvaluate["magic(4)"]
```

results in 

```
MATLink::errx: Undefined function 'magic' for input arguments of type 'double'.
```

The same happens with

```
MATLink`Engine`engEvaluate["magic(4)"]
```

so the problem is likely not in the part of MATLink written in Mma.

Reproduced on two different machines with Ubuntu 14.04.

A different Ubuntu 14.04 machine with Mathematica 9.0.1 and MATLAB R2013b does not show the problem.  Need to find out if this is caused by Mathematica 10 somehow.

Note: `mengine` was compiled with `-DMLITERFACE=3` and linked against ML64i3.

**Comment by szhorvat** [comment]
It's possible that this is caused by a corrupted path or working directory in MATLAB.

MATLink sets both on startup.

**Comment by szhorvat** [comment]
Fixed in MATLink 1.1.
